packages/autorest.python/autorest/codegen/models/response.py

```diff
@@ -59,6 +59,13 @@ def __init__(
         self.type = type
         self.nullable = yaml_data.get("nullable")
 
+    @property
+    def result_property(self) -> str:
+        field = self.yaml_data.get("resultProperty")
```

Comment:
do we still need this logic? i don't think we do since this was put in place to get around the bug that mark's looking into

Reply:
it is not bug but the guideline. Here is details: https://teams.microsoft.com/l/message/19:16f8f1ed70d044139be153707bdaee0e@thread.skype/1681897016100?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1681897016100&teamName=Azure%20SDK&channelName=TypeSpec%20(Cadl)%20Engineering&createdTime=1681897016100&allowXTenantAccess=false